newIDE/app/src/Utils/ZoomUtils.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+
+export const zoomStepBasePower = 1 / 16;
+
+const stepZoomFactor = Math.pow(2, 2 * zoomStepBasePower);
```

Comment:
Is there a reason for not using the power operator `**`?

Reply:
4ian once told me about the risk of having a navigator/browser that does not support this operator so I'm not using it anymore 😅 But I think that is more important for the runtime.
Better safe than sorry as they say